runtime/src/validate_block/tests.rs

```diff
@@ -135,23 +120,10 @@ fn build_block_with_proof(
 }
 
 #[test]
-fn validate_block_with_no_extrinsics() {
-	let (client, longest_chain) = create_test_client();
-	let parent_head = longest_chain.best_chain().expect("Best block exists");
-	let (block, witness_data) = build_block_with_proof(&client, Vec::new());
-	let (header, extrinsics) = block.deconstruct();
-
-	let block_data = ParachainBlockData::new(header.clone(), extrinsics, witness_data);
-
-	let res_header = call_validate_block(parent_head, block_data).expect("Calls `validate_block`");
-	assert_eq!(header, res_header);
-}
-
-#[test]
-fn validate_block_with_extrinsics() {
+fn validate_block_no_extrinsics() {
```

Comment:
That is not true.

And please also include some trivial balance transfers. That should also be part of the test framework.

Reply:
done in f6581e7